### PR TITLE
Initial changes. URL changes to griidc.org

### PR DESCRIPTION
**Pull request by fightingtexasaggie**

### CODE REVIEW

File: templates/hri-base.html.twig [hunk 1]
```diff
@@ -87,7 +87,7 @@
                                 <img src="{{ asset('build/images/hri-logo-min.svg') }}" alt="logo"
                                      class="mw-100 logo">
                             </a>
-                            <a class="navbar-brand" href="https://data.gulfresearchinitiative.org/" target="_blank">
+                            <a class="navbar-brand" href="https://griidc.org" target="_blank">
```
Comment (fightingtexasaggie) [inline templates/hri-base.html.twig:90]: Should this be hri.griidc.org here? I'm unsure...PO question.

File: templates/nas-grp-base.html.twig [hunk 2]
```diff
@@ -89,7 +89,7 @@
                                 <img src="{{ asset('build/images/nas-grp-logo.png') }}" alt="logo"
                                      style="width: 233px; margin-top: 5px" class="mw-100">
                             </a>
-                            <a class="navbar-brand" href="https://data.gulfresearchinitiative.org/" target="_blank">
+                            <a class="navbar-brand" href="https://griidc.org" target="_blank">
```
Comment (fightingtexasaggie) [inline templates/nas-grp-base.html.twig:92]: Similarly, should this be grp.griiidc.org here? Check with PO.

File: web/applications/metadata-editor-start/help.html [hunk 3]
```diff
@@ -27,7 +27,7 @@ <h3>Load from File:</h3>
 <li>This button will reload a blank form.
 </li></dd>
 <dd><h3>Check and Save to File:</h3>
-<li>This button will check that all required fields are completed then save the contents of the form to an ISO 19115-2 XML metadata file on your local computer.  This file is correctly formed XML adhering to the ISO 19115-2 Metadata standard but it is NOT validated at this stage.  Once saved, the metadata can be submitted to GRIIDC for validation through the Dataset Submission utility (<a href="https://data.gulfresearchinitiative.org/dataset-submission">https://data.gulfresearchinitiative.org/dataset-submission</a>)
+<li>This button will check that all required fields are completed then save the contents of the form to an ISO 19115-2 XML metadata file on your local computer.  This file is correctly formed XML adhering to the ISO 19115-2 Metadata standard but it is NOT validated at this stage.  Once saved, the metadata can be submitted to GRIIDC for validation through the Dataset Submission utility (<a href="https://griidc.org/dataset-submission">https://griidc.org/dataset-submission</a>)
```
Comment (fightingtexasaggie) [inline web/applications/metadata-editor-start/help.html:30]: Change this to data.griidc.org here. What's the status of MDE altogether?

File: web/applications/metadata-editor/help.html [hunk 4]
```diff
@@ -27,7 +27,7 @@ <h3>Load from File:</h3>
 <li>This button will reload a blank form.
 </li></dd>
 <dd><h3>Check and Save to File:</h3>
-<li>This button will check that all required fields are completed then save the contents of the form to an ISO 19115-2 XML metadata file on your local computer.  This file is correctly formed XML adhering to the ISO 19115-2 Metadata standard but it is NOT validated at this stage.  Once saved, the metadata can be submitted to GRIIDC for validation through the Dataset Submission utility (<a href="https://data.gulfresearchinitiative.org/dataset-submission">https://data.gulfresearchinitiative.org/dataset-submission</a>)
+<li>This button will check that all required fields are completed then save the contents of the form to an ISO 19115-2 XML metadata file on your local computer.  This file is correctly formed XML adhering to the ISO 19115-2 Metadata standard but it is NOT validated at this stage.  Once saved, the metadata can be submitted to GRIIDC for validation through the Dataset Submission utility (<a href="https://griidc.org/dataset-submission">https://griidc.org/dataset-submission</a>)
```
Comment (fightingtexasaggie) [inline web/applications/metadata-editor/help.html:30]: change to data.griidc.org here too

File: templates/Menus/griidc-main.html.twig
Comment (mickel1138) [inline templates/Menus/griidc-main.html.twig:1]: This file is technically no longer used in the new template.

File: templates/components/griidc-footer.html.twig
Comment (mickel1138) [inline templates/components/griidc-footer.html.twig:1]: Dont make ANY changes to this file, we'll do it in another PR.

File: templates/hri-base.html.twig (hunk 1 above)
Comment (mickel1138) [reply templates/hri-base.html.twig:90]: It's the GRIIDC logo, should go to GRIIDC. (.org)

File: templates/nas-grp-base.html.twig (hunk 2 above)
Comment (mickel1138) [reply templates/nas-grp-base.html.twig:92]: same as before, this is correct.

**Review by mickel1138 — APPROVED** [review]
LGTM
